stacks/upload-db-stack.js

```diff
@@ -30,6 +32,23 @@ export function UploadDbStack({ stack, app }) {
    */
   const uploadTable = new Table(stack, 'upload', uploadTableProps)
 
+  /**
+   * This table tracks space provisioning.
+   */
+  const provisionTable = new Table(stack, 'provision', provisionTableProps)
+
+  /**
+   * This table tracks metrics per space.
+   */
+  const delegationTable = new Table(stack, 'delegation', delegationTableProps)
+
+  const delegationBucket = new Bucket(stack, 'delegation-store', {
+    cors: true,
+    cdk: {
+      bucket: getBucketConfig('delegation', app.stage)
+    }
+  })
```

Comment:
this stack only includes databases, and I would keep it that way. We need to thing where the best place for this bucket is.

Reply:
ok - I've moved this out to `upload-api-stack.js` for now - happy to move it somewhere else but this seems like a decent default for now?

Reply:
I would likely move it to other place. So the reasoning is: we try to keep stacks with state separated from stateless stacks. Having it in `upload-api` adds a dependency that we cannot destroy the stack anymore and recreate.

Anyway, as far as I understand, we are getting rid of this right?

Reply:
ahh, got it - ok will find it a new home!

re: getting rid of it, good question - it looks like we may need to keep it around for legacy data, but we might be able to just point at R2 rather than move that data to S3 - in that case I'm not entirely sure how this configuration code would work - SST doesn't need to be responsible for creating and managing the bucket, so we should just need to pass some config in somewhere - let's find time to chat about how that would work!

Reply:
Yeah, if we want to just keep the R2 bucket, we just need to specify ENV Vars for the R2 bucket. You can see the replicator stack as we also use R2 buckets there

Reply:
ok - I've moved to creating this bucket the same way we do in the replicator stack: https://github.com/web3-storage/w3infra/pull/201